pyproject.toml

```diff
@@ -61,8 +61,8 @@ dev-civisml = [
 docs = [
     # docs/requirements.txt pins all transitive dependencies for a reproducible doc build.
     "numpydoc == 1.8.0",
-    "Sphinx == 7.4.7",
-    "sphinx-rtd-theme == 2.0.0",
+    "Sphinx == 8.1.3",
+    "sphinx-rtd-theme == 3.0.2",
```

Comment:
Updating to the latest releases for both [Sphinx](https://www.sphinx-doc.org/en/master/changes/8.0.html) and [sphinx-rtd-theme](https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html). No breaking changes from the major version bump for both packages should affect us. The one thing that caught my eye was sphinx-rtd-theme's deprecation of `html_theme_path`, which I've removed in `docs/conf.py` above.